packaging/opensc.spec

```diff
@@ -120,6 +120,10 @@ rm -rf %{buildroot}%{_mandir}/man1/pkcs11-register.1*
 rm %{buildroot}%{_datadir}/applications/org.opensc.notify.desktop
 rm %{buildroot}%{_mandir}/man1/opensc-notify.1*
 
+# remove the onepin pkcs11 module as it is only a symlink
+rm %{buildroot}%{_libdir}/pkcs11/onepin-opensc-pkcs11.so
+rm %{buildroot}%{_libdir}/onepin-opensc-pkcs11.so
```

Comment:
I specifically added the symlink to keep old configurations working, but feel free to remove it in the RPM, if you think that's not a problem in your case.

Reply:
The problem was how the symlink is created in when building the RPM. Its absolute symlink, which will point into the buildroot and will not work after the installation. I think it would be more suitable if the symlink would be relative to the original file, what do you think?

Reply:
The linked page propose to use only
```
        cd $(DESTDIR)$(bindir) && \
          mv -f prog$(EXEEXT) prog-$(VERSION)$(EXEEXT) && \
          $(LN_S) prog-$(VERSION)$(EXEEXT) prog$(EXEEXT)
```
> Note that we rename the program so that a new version will erase the symbolic link, not the real binary. **Also we cd into the destination directory in order to create relative links.**

which is relative symlink while you use hard-links. I did not expect that to be an issue, but with this guideline, it is something that we should change too. I will update the PR as I think it is not worth separate PR.